buildconfig/pygame-stubs/__init__.pyi

```diff
@@ -69,4 +69,6 @@ def encode_file_path(
     obj: Any, etype: Optional[Type[Exception]] = UnicodeEncodeError
 ) -> bytes: ...
 def register_quit(callable: Callable) -> None: ...
+def get_screensaver_enabled() -> bool: ...
```

Comment:
Nice changes, didn't know it is even possible to do something like this with SDL :+1:
just a note can you move this 2 functions from \_\_init\_\_.pyi to display.pyi (because with this it would mean that you call this functions with `pygame.get_screensaver_enabled()`, and as far as I have understood it should be called with `pygame.display.get_screensaver_enabled()`

Reply:
Thanks for catching that.  I intended to use `display.pyi` but apparently managed to write it in the wrong file. :)